src/Catalog/Dnx/DnxCatalogCollector.cs

```diff
@@ -70,35 +70,58 @@ public class DnxCatalogCollector : CommitCollector
             var destinationStorage = _storageFactory.Create(id);
             var destinationUri = destinationStorage.GetUri(DnxMaker.GetRelativeAddressNupkg(id, version));
 
-            if (await destinationStorage.AreSyncronized(sourceUri, destinationUri))
+            var isNupkgSynchronized = await destinationStorage.AreSyncronized(sourceUri, destinationUri);
```

Comment:
>AreSyncronized [](start = 63, length = 14)

Could you rename this method: `Syncronized` -> `Synchronized` :)

Reply:
Sure

Reply:
Done.